wagtailio/services/models.py

```diff
@@ -0,0 +1,24 @@
+from django.db import models
+from wagtail.admin import edit_handlers
+from wagtail.core import blocks, fields
+from wagtail.core.models import Page
+
+from wagtailio.services.blocks import SectionBlock
+
+
+class ServicesPage(Page):
+    intro = models.TextField(null=True, blank=True)
+    body = fields.StreamField(
+        blocks.StreamBlock(
+            [('section', SectionBlock())],
+            min_num=4,
+            max_num=4,
+        ),
+        null=True,
```

Comment:
**Suggestion**: I'm not sure about the effect of `null=True` on a StreamField. Underneath, it's a text field, and omitting `null=True` works fine: the default value is something like `"[]"`.

Reply:
I believe that was a left over from me adding the fields one by one and I could not create the migrations without allowing `null`. But since I recreated a single migration after the fact, this should not be necessary any more. 

I will remove it.